README.md

```diff
@@ -6,25 +6,17 @@
 
 ### getting a public photo
 ```
```

Comment:
Could you tack a `js` on after the backticks? That’ll get JS-aware syntax highlighting.